Panic when piping `helm help`

The `helm` client errors in this case with "broken pipe." Other `helm` commands don't seem to have this problem.
```go
$ helm help | head -5
NAME:
   helm - The Kubernetes package manager

To begin working with Helm, run the 'helm update' command:

panic: template: help:15:30: executing "help" at <
GLOBAL OPTIONS:
   >: write /dev/stdout: broken pipe

goroutine 1 [running]:
github.com/helm/helm/vendor/github.com/codegangsta/cli.printHelp(0xe5d1e8, 0xc82002c010, 0x9e5b60, 0x23d, 0x7e4e40, 0xc8201a0d80)
	/Users/matt/Projects/src/github.com/helm/helm/vendor/github.com/codegangsta/cli/help.go:183 +0x40a
github.com/helm/helm/vendor/github.com/codegangsta/cli.ShowAppHelp(0xc8201a0fc0)
	/Users/matt/Projects/src/github.com/helm/helm/vendor/github.com/codegangsta/cli/help.go:110 +0x7b
github.com/helm/helm/vendor/github.com/codegangsta/cli.glob.func1(0xc8201a0fc0)
	/Users/matt/Projects/src/github.com/helm/helm/vendor/github.com/codegangsta/cli/help.go:81 +0x161
github.com/helm/helm/vendor/github.com/codegangsta/cli.Command.Run(0x8576d8, 0x4, 0x0, 0x0, 0xc629a0, 0x1, 0x1, 0x942be0, 0x30, 0x0, ...)
	/Users/matt/Projects/src/github.com/helm/helm/vendor/github.com/codegangsta/cli/command.go:131 +0x1055
github.com/helm/helm/vendor/github.com/codegangsta/cli.(*App).Run(0xc8201a0d80, 0xc82000a240, 0x2, 0x2, 0x0, 0x0)
	/Users/matt/Projects/src/github.com/helm/helm/vendor/github.com/codegangsta/cli/app.go:175 +0xfd6
github.com/helm/helm/vendor/github.com/codegangsta/cli.(*App).RunAndExitOnError(0xc8201a0d80)
	/Users/matt/Projects/src/github.com/helm/helm/vendor/github.com/codegangsta/cli/app.go:186 +0x54
main.main()
	/Users/matt/Projects/src/github.com/helm/helm/helm.go:6 +0x25

goroutine 17 [syscall, locked to thread]:
runtime.goexit()
	/usr/local/Cellar/go/1.5.3/libexec/src/runtime/asm_amd64.s:1721 +0x1

goroutine 5 [chan receive]:
github.com/helm/helm/vendor/github.com/golang/glog.(*loggingT).flushDaemon(0xc72aa0)
	/Users/matt/Projects/src/github.com/helm/helm/vendor/github.com/golang/glog/glog.go:879 +0x67
created by github.com/helm/helm/vendor/github.com/golang/glog.init.1
	/Users/matt/Projects/src/github.com/helm/helm/vendor/github.com/golang/glog/glog.go:410 +0x297
```

--- a/glide.lock
+++ b/glide.lock
@@ -1,5 +1,5 @@
-hash: ca470e3495e875ff100e6e886ee686e835d2862c2090a1a55e9d3b5005ca6650
-updated: 2016-04-11T11:21:50.997021805-04:00
+hash: 54848c15ec687202a4a144233bad37eec7cf06476df5cbe46b2c54724f4b32ec
+updated: 2016-04-20T10:26:47.634590058-06:00
 imports:
 - name: bitbucket.org/ww/goautoneg
   version: 75cd24fc2f2c
@@ -17,7 +17,7 @@ imports:
 - name: github.com/cloudfoundry-incubator/candiedyaml
   version: 5cef21e2e4f0fd147973b558d4db7395176bcd95
 - name: github.com/codegangsta/cli
-  version: f445c894402839580d30de47551cedc152dad814
+  version: 71f57d300dd6a780ac1856c005c4b518cfd498ec
 - name: github.com/davecgh/go-spew
   version: 3e6e67c4dcea3ac2f25fd4731abc0e1deaf36216
   subpackages:
@@ -68,7 +68,7 @@ imports:
 - name: github.com/Masterminds/sprig
   version: 679bb747f11c6ffc3373965988fea8877c40b47b
 - name: github.com/Masterminds/vcs
-  version: b22ee1673cdd03ef47bb0b422736a7f17ff0648c
+  version: fa85cceafacd29c84a8aa6e68967bb9f1754e5e3
 - name: github.com/matttproud/golang_protobuf_extensions
   version: fc2b8d3a73c4867e51861bbdd5ae3c1f0869dd6a
   subpackages:
@@ -96,7 +96,7 @@ imports:
   version: ca42424d18c76d0d51a4cccd830d11878e9e5c17
   repo: https://github.com/coreos/gexpect
 - name: golang.org/x/crypto
-  version: 3fbbcd23f1cb824e69491a5930cfeff09b12f4d2
+  version: 3305186468e57f6c7535c9e97cb0bde4b2b65908
   subpackages:
   - ssh/terminal
   - nacl/box

--- a/glide.yaml
+++ b/glide.yaml
@@ -1,7 +1,7 @@
 package: github.com/helm/helm
 import:
 - package: github.com/codegangsta/cli
-  version: f445c894402839580d30de47551cedc152dad814
+  version: 71f57d300dd6a780ac1856c005c4b518cfd498ec
 - package: github.com/deis/pkg
   subpackages:
   - prettyprint

